### DISCUSSION TITLE
how can i keep in current month when select range end

### DISCUSSION THREAD

**Discussion by Dawnwangzi** [wojtekmaj/react-calendar#639] [Q&A] [answered]
1. set selectRange = true, showDoubleView = false
3. click 2022/6/1
4. turn to page July
5. click 2022/7/15
it auto jump to page June
how can it keep in July after the second click

**Comment by wojtekmaj** [comment, marked as answer]
You can't, but I definitely do agree there should be an option to do so. So, in da85576, I have added `goToRangeStartOnSelect` prop (set to `true` by default for now to avoid breaking changes), that you can set to `false` to achieve the desired behavior.

**Reply by Dawnwangzi** [reply]
can you release a new version? thank you.

**Reply by wojtekmaj** [reply]
There you go

**Reply by Dawnwangzi** [reply]
i update it to 3.9.0, but when i use goToRangeStartOnSelect, i got an ERROR.

Type '{ value: [Date, Date]; selectRange: true; maxDate: Date; minDate: Date; minDetail: "month"; showNeighboringMonth: false; prev2Label: null; next2Label: null; nextLabel: Element; ... 5 more ...; onClickDay: (value: any, event: any) => void; }' is not assignable to type 'IntrinsicAttributes & CalendarProps'.
  Property 'goToRangeStartOnSelect' does not exist on type 'IntrinsicAttributes & CalendarProps'.

maybe @types/react-calendar should be update.

**Reply by Dawnwangzi** [reply]
hi ~ i try 'goToRangeStartOnSelect false' , but it doesn't work.
you can see in this link.
https://codesandbox.io/s/laughing-yalow-9upyht?file=/src/App.js

**Reply by lemontini** [reply]
I can confirm (@Dawnwangzi) - the feature doesn't work.

**Reply by wojtekmaj** [reply]
React-Calendar is now natively TypeScript and does not require external types, so this problem will not occur.

**Reply by peterkulacs** [reply]
@wojtekmaj can you please check this? `goToRangeStartOnSelect={false}` doesn't seem to work with the latest version.
